pkgs/development/ocaml-modules/kafka/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib, dune, base, cmdliner, ocaml_lwt,
+  rdkafka, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "ocaml${ocaml.version}-kafka";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "didier-wenzek";
+    repo = "ocaml-kafka";
+    rev = version;
+    sha256 = "0lb8x0wh7sf8v9mjwhq32azjz54kw49fsjfb7m76z4nhxfkjw5hy";
+  };
+
+  buildInputs = [ ocaml findlib dune base cmdliner ocaml_lwt rdkafka zlib ];
+
+  propagatedBuildInputs = [ rdkafka zlib ];
```

Comment:
The libraries that appear in `propagatedBuildInputs` need not be duplicated in the `buildInputs` list.